### PR TITLE
Fix release build through command line errors

### PR DESCRIPTION
**Pull request by kunall17**
Now the react-native run-ios --configuration Release works now, without any changes

**Requires test from multiple people to make sure it works for everyone and not only my setup!**

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@kunall17, it looks like you've already signed the Dropbox CLA. Thanks!

File: ios/ZulipMobile.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1191,7 +1249,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "export SENTRY_PROPERTIES=sentry.properties\nexport NODE_BINARY=node\n../node_modules/sentry-cli-binary/bin/sentry-cli react-native xcode ../node_modules/react-native/scripts/react-native-xcode.sh";
+			shellScript = "if [[ $SDKROOT == *\"iPhoneSimulator10\"* ]]; then\necho [Bundle React Native code and images] Skipping sentry\n../node_modules/react-native/scripts/react-native-xcode.sh\nelse\nexport SENTRY_PROPERTIES=sentry.properties\nexport NODE_BINARY=node\n../node_modules/sentry-cli-binary/bin/sentry-cli react-native xcode ../node_modules/react-native/scripts/react-native-xcode.sh\nfi";
```
Comment (borisyankov) [inline ios/ZulipMobile.xcodeproj/project.pbxproj]: Will not that be version 10 specific regex?

File: ios/sentry.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
-defaults.url=https://sentry.io
+defaults.url=https://sentry.io/
 defaults.org=zulip
 defaults.project=zulip-mobile
-auth.token=ADD-DSN-HERE
+auth.token=ADD-TOKEN-HERE
```
Comment (borisyankov) [inline ios/sentry.properties]: ?

File: src/ZulipMobile.js [hunk 3]
```diff
@@ -10,7 +11,7 @@ import config from './config';
 
 require('./i18n/locale');
 
-if (config.enableSentry) {
+if (config.enableSentry && !DeviceInfo.isEmulator()) {
```
Comment (borisyankov) [inline src/ZulipMobile.js]: You can add this logic to 'config.enableSentry'.

Comment (kunall17) [inline src/ZulipMobile.js]: Getting this error in unit test's if I use
> import DeviceInfo from 'react-native-device-info';

in the config.js
```
    TypeError: _reactNativeDeviceInfo2.default.isSimulator is not a function
      
      at Object.<anonymous> (src/config.js:5:76)
      at Object.<anonymous> (src/api/checkCompatibility.js:2:13)
      at Object.<anonymous> (src/api/index.js:2:37)
```

File: ios/sentry.properties (hunk 2 above)
Comment (kunall17) [inline ios/sentry.properties]: As specified https://github.com/getsentry/react-native-sentry/issues/140#issuecomment-318572188 here its auth token there and not dsn!

File: ios/ZulipMobile.xcodeproj/project.pbxproj (hunk 1 above)
Comment (kunall17) [inline ios/ZulipMobile.xcodeproj/project.pbxproj]: Done!

**Comment by kunall17** [comment]
Btw, this fixes the `react-native run-ios` debug configuration as well! :)

**Comment by Sam1301** [comment]
this fixes the error in release and debug build for me 👍

**Comment by borisyankov** [comment]
Looks good. Thanks. 
That is quite an unblocking fix.

**Pull request merged**